docs/discussion.md

```diff
@@ -0,0 +1,205 @@
+# **Interpolation**
+
+## **How the btwxt library works**
+## Big Ladder Software
+
+## 1-D Linear
```

Comment:
Should there be a hyphen between the "1" and "D"? It looks funny but I guess we'd want it for "N-D"...